vspc/server.py

```diff
@@ -214,7 +214,6 @@ async def rewrite_file(self, path, data):
         tmp_path = "tmp_" + path
         async with aiofiles.open(tmp_path, "wb") as f:
             await f.write(data)
-        await aio_os.remove(path)
         await aio_os.rename(tmp_path, path)
```

Comment:
At least for Python's `os`, `rename()` doesn't overwrite. Instead, the [docs](https://docs.python.org/3/library/os.html#os.rename) talk about using `replace()`. Could you link the aiofile docs here/in the commit for future reference?

Reply:
from docs link `rename` -> `If both are files, dst it will be replaced silently if the user has permission.`
`replace` is doing the same but it's a cross-platform operation -> `If you want cross-platform overwriting of the destination, use replace().`

Reply:
You're right. I only read until "will raise OSError" and missed the "in a number of cases". :+1: